src/lpython/parser/parser.yy

```diff
@@ -678,8 +678,8 @@ expr_list_opt
     ;
 
 expr_list
-    : expr_list "," expr %prec FOR { $$ = $1; LIST_ADD($$, $3); }
-    | expr %prec FOR { LIST_NEW($$); LIST_ADD($$, $1); }
+    : expr_list "," expr %prec "not" { $$ = $1; LIST_ADD($$, $3); }
+    | expr %prec "not" { LIST_NEW($$); LIST_ADD($$, $1); }
```

Comment:
Why change `FOR` to `"not"`, any specific reason?

Reply:
> 1. Using precedence for `FOR` in `expr_list` caused expressions using `in` to be considered as for-statements.
> 2. Precedence has been given to `not` now in expr_list, and the `FOR` precedence is unused (I didn't remove it since there are chances that it can be used later)

I see.